### PR TITLE
gets list of all articles

### PR DESCRIPTION
**Pull request by andmikpar**

### CODE REVIEW

File: src/components/Articles.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+import { getAllArticles } from '../utils/api';
+
+const Articles = () => {
+  const [articleList, setArticleList] = useState([]);
+
+  useEffect(() => {
+    getAllArticles().then((articleData) => {
+      setArticleList(articleData);
+    });
```
Comment (popatre) [inline src/components/Articles.jsx]: Need to handle the loading here as well

**Review by popatre — CHANGES_REQUESTED** [review]
Could you add some loading state to let us know something is happening?

File: src/components/Articles.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        setIsError(err.code);
+        setIsLoading(false);
+      });
+  }, []);
+
+  return (
+    <div className="articles">
+      {articleList.map((article) => {
+        return (
+          <div className="article">
```
Comment (B7Bprog) [inline src/components/Articles.jsx]: You could add a 'key' attribute to this tag with the article_id for example to let react to track it. It still works without doing so though.

**Review by B7Bprog — APPROVED** [review]
Great work Andy, responsive design, great visual feedback of loading pages, good use of utils functions for API requests. Please consider my other comment, then merge. 🎉

**Pull request merged**